client/rest-client/src/main/java/org/infinispan/client/rest/RestCacheClient.java

```diff
@@ -337,6 +337,16 @@ default CompletionStage<RestResponse> query(String query) {
     */
    CompletionStage<RestResponse> query(String query, int maxResults, int offset);
 
+   /**
+    * Executes an Ickle-query
+    *
+    * @param query      the ickle query
+    * @param maxResults the maximum number of results to return
+    * @param offset     the offset within the result from which to return results
+    * @param hitCountAccuracy
```

Comment:
What does this variable do?

Reply:
I am guessing it means we can have a lowered accuracy to return sooner?

Reply:
I was also implying to update the method documentation here :)

Reply:
Oh! I added it :)